app/src/main/res/layout/activity_view_habit.xml

```diff
@@ -0,0 +1,164 @@
+        android:background="@color/cardview_shadow_end_color"/>
+
+    <TextView
+        android:id="@+id/daysHeading"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/dates"
+        android:text="On what days will you work on this habit?"
+        android:layout_marginLeft="10dp"
+        android:layout_marginTop="20dp"
+        android:background="@color/cardview_shadow_end_color"/>
+
+    <ca.antonious.materialdaypicker.MaterialDayPicker
+        android:id="@+id/days"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/daysHeading"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:background="@color/cardview_shadow_end_color"/>
+
+    <TextView
+        android:id="@+id/reasonHeading"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/days"
+        android:text="Reason"
+        android:layout_marginLeft="10dp"
+        android:layout_marginTop="20dp"
+        android:background="@color/cardview_shadow_end_color"/>
+
+    <TextView
+        android:id="@+id/reason"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/reasonHeading"
+        android:text="I want to work on this because...."
+        android:textSize="20dp"
+        android:textColor="@color/black"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:background="@color/cardview_shadow_end_color"/>
+
+    <TextView
+        android:id="@+id/progressHeading"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/reason"
+        android:text="Progress"
+        android:layout_marginLeft="10dp"
+        android:layout_marginTop="20dp"
+        android:background="@color/cardview_shadow_end_color"/>
+
+    <androidx.core.widget.ContentLoadingProgressBar
+        android:id="@+id/progress"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/progressHeading"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp" />
+
+    <Switch
+        android:id="@+id/privateSwitch"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/progress"
+        android:layout_marginTop="20dp"
+        android:layout_marginRight="340dp" />
+
+    <TextView
+        android:id="@+id/privateSwitchText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/progress"
+        android:text="Switch to private"
+        android:layout_marginTop="20dp"
+        android:layout_marginLeft="60dp"/>
```

Comment:
```suggestion
    <Switch
        android:id="@+id/privateSwitch"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_below="@+id/progress"
        android:layout_marginRight="10dp"
        android:layout_marginLeft="10dp"
        android:text="Switch to private" />
```
I integrated the text into the switch, so it looks less crowded. Let me know if this is okay.
![image](https://user-images.githubusercontent.com/38046772/140466793-bd4e0409-44f9-4268-a4c0-bbdf5fb2d953.png)